src/apphosting/yaml.spec.ts

```diff
@@ -1,195 +1,52 @@
 import { expect } from "chai";
 import { AppHostingYamlConfig } from "./yaml";
 
-describe("yaml", () => {
-  describe("environment variables", () => {
-    let apphostingYaml: AppHostingYamlConfig;
-    beforeEach(() => {
-      apphostingYaml = AppHostingYamlConfig.empty();
-    });
-
-    it("adds environment variables and retrieves them correctly", () => {
-      apphostingYaml.addEnvironmentVariable({
-        variable: "TEST_1",
-        value: "value_1",
-      });
-
-      apphostingYaml.addEnvironmentVariable({
-        variable: "TEST_2",
-        value: "value_2",
-      });
-
-      expect(JSON.stringify(apphostingYaml.environmentVariables)).to.equal(
-        JSON.stringify([
-          {
-            variable: "TEST_1",
-            value: "value_1",
-          },
-          {
-            variable: "TEST_2",
-            value: "value_2",
-          },
-        ]),
-      );
-    });
-
-    it("overwrites stored environment variable if another is added with same name", () => {
-      apphostingYaml.addEnvironmentVariable({
-        variable: "TEST",
-        value: "value",
-      });
-
-      apphostingYaml.addEnvironmentVariable({
-        variable: "TEST",
-        value: "overwritten_value",
-      });
-
-      expect(JSON.stringify(apphostingYaml.environmentVariables)).to.equal(
-        JSON.stringify([{ variable: "TEST", value: "overwritten_value" }]),
-      );
+describe("merge", () => {
+  it("merges incoming apphosting yaml config with precendence", () => {
```

Comment:
```suggestion
  it("merges incoming apphosting.yaml config with precendence", () => {
```

Reply:
Technically incorrect, no? It's about merging apphosting.local.yaml over apphosting.emulators.yaml over apphosting.yaml